utils/python/CIME/SystemTests/README

```diff
@@ -1,11 +1,13 @@
 The following are the test functionality categories:
   1) smoke tests
-  2) restart tests
-  3) threading/pe-count modification tests
-  4) sequencing (layout) modification tests
-  5) multi-instance tests
-  6) performance tests
-  7) spinup tests (TODO)
+  2) basic reproducibility tests
```

Comment:
Isn't the scripts/Testing/Testcases directory (which contains config_tests.xml) going away?  I'd remove it from config_tests.xml since there is a README in that directory with a pointer to the README here.